### PR TITLE
Unit Testing: React Testing Library

### PR DESCRIPTION
**Pull request by wwwdrt**
# 👋 Math Magicians: React Testing Library

> I'm looking forward to your feedback 🚀

Please feel free to reach out to me on Slack anytime. You can send me a direct message or give me a call. If you choose to call, please introduce yourself so I know who I'm speaking with. I'm here to help, so don't hesitate to get in touch 😁

<h3 align=center>⬇️</h3>

<div align="center">
  <a target="_blank" href="https://t.me/apexsec">
    Telegram
  </a>
<a target="_blank" href="https://your-slack-link">
  Slack
</a>
<a target="_blank" href="https://your-zoom-link">
  Zoom
</a>
</div>

<br>

<div align="center">
  

### Latest project 🔥"[Ruby on Rails Blog](https://github.com/wwwdrt/rails_blog)"
</div>

# 📖 Getting Started

> In this part of the project I wrote unit tests for all of the components using react testing library.

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @wwwdrt ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] You have submitted a project missing commits from your pair programming group. Kindly ensure your team members from your pair programming group commit to this pull request before you resubmit for review.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Quotes.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    render(<Quotes />);
+
+    expect(fetch).toHaveBeenCalledTimes(1);
+    expect(fetch).toHaveBeenCalledWith('https://api.api-ninjas.com/v1/quotes?category=money', {
+      headers: {
+        'X-Api-Key': 'Q2raRaEu8SFowxGsBjYHlQ==hGwk854gekKZLv2a',
+      },
+    });
+  });
+});
```
Comment (abdulhamiid) [inline src/Components/Quotes.test.js:30]: The test passes but I get the following warning in `src\Components\Quotes.js`:
image

Kindly fix that to ensure that the app is running smoothly as it should  🛠💪.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️Write unit tests with React Testing Library.
- ✔️ Write unit tests with snapshots with Jest.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi team 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ All tests are passing, perfect!
-  ✅ Great work using the right Gitflow 
-  ✅ Warning is cleared, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**